src/test/regress/expected/multi_create_table_constraints.out

```diff
@@ -41,15 +32,15 @@ CREATE TABLE pk_on_non_part_col
 	other_col integer PRIMARY KEY
 );
 SELECT master_create_distributed_table('pk_on_non_part_col', 'partition_col', 'hash');
-ERROR:  cannot distribute relation: "pk_on_non_part_col"
+ERROR:  cannot create constraint on "pk_on_non_part_col"
```

Comment:
We check the constraints of CREATE TABLE and ALTER TABLE statements using the same part of the code. For the ALTER TABLE case, "cannot distribute relation" seems meaningless that the table has already distributed.